src/state/slices/portfolioSlice/selectors.ts

```diff
@@ -986,23 +986,48 @@ export const selectAssetEquityItemsByFilter = createDeepEqualOutputSelector(
       }
     })
     const staking = stakingOpportunities.map(stakingOpportunity => {
-      // Because the underlying assets can have different precisions
-      // We need to convert it to the asset we are viewing to get correct amounts to sum together.
-      const underlyingAssetPrecision =
-        assets[stakingOpportunity.assetId]?.precision ?? asset?.precision
-      const cryptoAmountBaseUnit = convertPrecision({
-        value: bnOrZero(stakingOpportunity.cryptoAmountBaseUnit),
-        inputExponent: underlyingAssetPrecision,
-        outputExponent: asset?.precision ?? 0,
-      }).toString()
-      const amountCryptoPrecision = fromBaseUnit(
-        bnOrZero(cryptoAmountBaseUnit),
-        asset?.precision ?? 0,
-      )
+      const { amountCryptoPrecision, fiatAmount } = (() => {
```

Comment:
```suggestion
      const { amountCryptoPrecision, amountUserCurrency } = (() => {
```

Reply:
Fixed in [4bade8f](https://github.com/shapeshift/web/pull/7644/commits/4bade8f9fe1ac3742ff2474ea6faa38eed2ad30e)